media-fonts/jetbrains_mono/jetbrains_mono-1.0.3.recipe

```diff
@@ -0,0 +1,37 @@
+SUMMARY="A monospace font designed for programmers"
+DESCRIPTION="JetBrains Mono’s typeface forms are simple and free \
+from unnecessary details. Rendered in small sizes, the text looks crisper.
+
+The easier the forms, the faster the eye perceives them \
+and the less effort the brain needs to process them."
+HOMEPAGE="https://www.jetbrains.com/lp/mono/"
+COPYRIGHT="2020 Philipp Nurullin \
+	2020 Konstantin Bulenkov \
+	The whole JetBrains Team"
```

Comment:
Not completely sure here (COPYRIGHT):
https://github.com/JetBrains/JetBrainsMono#credits

Reply:
The repository does contain code from the two authors, which is licensed under the Apache license.

I'll open an issue on their repository just to make sure.

e.g. https://github.com/JetBrains/JetBrainsMono/blob/master/scripts/UpdateMetaData.kt

Reply:
Proposed a relevant change to your repository, @Begasus.

Reply:
Also opened this issue, just to be sure: https://github.com/JetBrains/JetBrainsMono/issues/138

Reply:
Thanks :)

Reply:
That's fine